pull_request_template.md

```diff
@@ -1,13 +1,6 @@
 ### What this PR does (required):
-- list PR functionality here
+The Landing Page have being created, which is displayed when the app is run.
```

Comment:
Please don't edit the PR template, you can edit the description of the PR instead